content/5_Plat/RURQ.mdx

```diff
@@ -221,9 +221,165 @@ queries.
 	</Resource>
 </Resources>
 
-### Implementation
+### Solution - Range Updates & Sums
 
-<IncompleteSection />
+<Spoiler title="Solution">
+
+This question asks you to support the following types of queries. 
+
+-	Add a value to all elements within the range $[a,b]$.
+
+-	Set all values within the range $[a,b]$ to a certain value. 
+
+-	Find the sum of all values in range $[a,b]$.
+
+Consider the first two types of queries, a lazy tag will be created in each node of the tree for each type. In this solution, $lzAdd$ will represent the lazy tag for the range add query and $lzSet$ will represent the lazy tag for the range set query. 
+
+Given the two different update queries, a total of four different situations might take place as a result of any update. 
+
+-	Range add ($lzAdd$) when $lzSet$ equals 0: Simply add the new value to the pre-existing value. 
+
+-	Range add ($lzAdd$) when $lzSet$ doesn't equal to 0: Add the new value to $lzSet$ and clear $lzAdd$. 
```

Comment:
> doesn't equal to 0

still hasn't been changed

Reply:
> > doesn't equal to 0
> 
> still hasn't been changed

Didn't u say that "doesn't equal to 0" is ok though? What should I change it to if this is not okay ?

Reply:
"doesn't equal 0" was ok, "doesn't equal to 0" is not